tests/Base.nut

```diff
@@ -0,0 +1,81 @@
+// MIT License
```

Comment:
Because the test functions that are in the class inherited from the class, which in turn is inherited from the ImpTestCase class, are not executed by imptest.